sdk/Lusid.Sdk/Utilities/LusidApiFactory.cs

```diff
@@ -68,6 +68,15 @@ public LusidApiFactory(Configuration configuration)
 
         private static Dictionary<Type, IApiAccessor> Init(Configuration configuration)
         {
+            // If some retry policy has already been assigned, use it.
+            // Users can combine their own policy with the default policy by using the .Wrap() method.
```

Comment:
Can you provide an example of this in your tests?

Reply:
I'll try to find a way to make an example test using this

Reply:
Done